integrations/unity-csharp/Assets/VDK/VDKUserInfoEditor.cs

```diff
@@ -0,0 +1,143 @@
+﻿using UnityEngine;
+using System;
+using UnityEditor;
+using UnityEditor.PackageManager;
+using UnityEditor.PackageManager.Requests;
+using Vault;
+
+
+
+
```

Comment:
So much whitespace

Reply:
Ahh cheers, I'll make sure to clear that out in future work ty